src/main/app/case/case.ts

```diff
@@ -212,6 +212,9 @@ export interface Case {
   op_courtProceedingsOrders?: C100OrderTypes[];
   op_otherProceedings?: OtherProceedings;
   c1A_safteyConcerns?: C1ASafteyConcerns;
+  c1A_previousAbductionsShortDesc?: string;
```

Comment:
@niranjan-301  Are we using these fields in any decision making or navigation ?
If not please remove them, they will added to userCase in session by default.

Reply:
done